java/src/jmri/jmrit/dispatcher/DispatcherFrame.java

```diff
@@ -178,7 +178,7 @@ public int loadTrainFromTrainInfo(String traininfoFileName) {
      */
     public int loadTrainFromTrainInfo(String traininfoFileName, String overRideType, String overRideValue) {
         //read xml data from selected filename and move it into trainfo
-        try {
+//        try {
```

Comment:
Not too sure why this would be removed with catching and handling errors deeper in the code.

Reply:
try catch should be reinstated

Reply:
I reinstated with a commit last night. Let me know if there are any problems

Reply:
The try catch is still commented out.